portfolio.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<html lang="en">
+		<link rel="stylesheet" type="text/css" href="normalize.css">
```

Comment:
Doesn't look like you've got your normalise.css being tracked by git at the moment, but I assume it's in the right directory and all seems to be working?